Assets/Scripts/Controllers/HeaderController.cs

```diff
@@ -38,30 +62,92 @@ public class HeaderController : MonoBehaviour {
 				headerTitle.text = Config.CurrentlyLookingAtText;
 				headerValue.text = nodeController.HighlightedNode.Title;
 			} else {
-				headerTitle.text = "";
-				headerValue.text = "";
+				headerTitle.text = string.Empty;
+				headerValue.text = string.Empty;
 			}
 			if (nodeController.SelectedNode != null) {
 				if (nodeController.HighlightedNode != null && nodeController.HighlightedNode != nodeController.SelectedNode) {
 					headerTitle.text = Config.CurrentlyLookingAtText;
 					headerValue.text = nodeController.HighlightedNode.Title;
+					ShowOnlyConnectionsCount(connectionController.GetNodeNeighbours(nodeController.HighlightedNode).ToArray().Length);
 				} else {
 					headerTitle.text = Config.CurrentlySelectedText;
 					headerValue.text = nodeController.SelectedNode.Title;
+					ShowOnlyConnectionsCount(null);
+				}
+			} else {
+				connectionController.OnConnectionRangeChanged?.Invoke(0, 0, 0);
+				ShowOnlyConnectionsCount(connectionController.GetNodeNeighbours(nodeController.HighlightedNode).ToArray().Length);
```

Comment:
And if ```HighlightedNode``` is null...
![image](https://user-images.githubusercontent.com/8643919/68073394-13c05300-fd90-11e9-90ea-de141b47a7e3.png)

Reply:
Ups, that conditional statement is getting longer and longer :D